include/xmlwrapp/node.h

```diff
@@ -892,18 +892,14 @@ class XMLWRAPP_API node
 
 // Comparison operators for xml::node iterators
 
-inline bool XMLWRAPP_API operator==(const node::iterator& lhs,
-                                    const node::iterator& rhs)
+inline bool operator==(const node::iterator& lhs, const node::iterator& rhs)
     { return lhs.get_raw_node() == rhs.get_raw_node(); }
-inline bool XMLWRAPP_API operator!=(const node::iterator& lhs,
-                                    const node::iterator& rhs)
+inline bool operator!=(const node::iterator& lhs, const node::iterator& rhs)
     { return !(lhs == rhs); }
 
-inline bool XMLWRAPP_API operator==(const node::const_iterator& lhs,
-                                    const node::const_iterator& rhs)
+inline bool operator==(const node::const_iterator& lhs, const node::const_iterator& rhs)
     { return lhs.get_raw_node() == rhs.get_raw_node(); }
-inline bool XMLWRAPP_API operator!=(const node::const_iterator& lhs,
-                                    const node::const_iterator& rhs)
+inline bool operator!=(const node::const_iterator& lhs, const node::const_iterator& rhs)
```

Comment:
I do not understand the removal of `XMLWRAPP_API` here and in the other place. What warnings does the part of commit message that refers to this, i.e. "remove a couple of superfluous ones, as this resulted in -Wattributes from gcc"  mean?

Don't the inline symbols still need to exist in the shared library binary (and so be appropriately marked), because the compiler may decide not to inline them?

Reply:
I think you're right and I've made a mistake because I can't reproduce these warnings with `__declspec(dllexport)` any more, but I can reproduce them when using `__declspec(dllimport)` (which indeed doesn't make sense, of course, and gcc is perfectly right to warn about those). So I must have done some mistake in `export.h` which resulted in a wrong declspec being used, sorry for not realizing this and thanks for noticing it!

I've force-pushed the fixed version now.